packages/expo-screen-orientation/ios/EXScreenOrientation/EXScreenOrientationRegistry.m

```diff
@@ -25,14 +26,24 @@ - (instancetype)init
     _moduleInterfaceMasks =  [NSMapTable weakToStrongObjectsMapTable];
     _notificationListeners = [NSPointerArray weakObjectsPointerArray];
     _currentTraitCollection = nil;
+    _currentScreenOrientation = 0;
+    _lastOrientationMask = 0;
     
     [[NSNotificationCenter defaultCenter] addObserver:self
                                             selector:@selector(handleDeviceOrientationChange:)
                                                 name:UIDeviceOrientationDidChangeNotification
                                               object:[UIDevice currentDevice]];
-    
+    UM_WEAKIFY(self);
     dispatch_async(dispatch_get_main_queue(), ^{
       [[UIDevice currentDevice] beginGeneratingDeviceOrientationNotifications];
+      
+      UM_ENSURE_STRONGIFY(self)
+      if (@available(iOS 13, *)) {
+        self.currentScreenOrientation = UIApplication.sharedApplication.windows[0].windowScene.interfaceOrientation;
+      } else {
+        // statusBarOrientation was deprecated in iOS 13
+        self.currentScreenOrientation = UIApplication.sharedApplication.statusBarOrientation;
+      }
```

Comment:
Shouldn't we call it on the main queue **synchronously**? `dispatch_async` means that this may be executed some time from now, maybe it would be wrong to other methods be executed before assigning the `currentScreenOrientation` initial value?